src/portable/renesas/usba/dcd_usba.c

```diff
@@ -230,7 +249,7 @@ static unsigned select_pipe(unsigned num, unsigned attr)
 {
   USB0.PIPESEL.WORD  = num;
   USB0.D0FIFOSEL.WORD = num | attr;
-  while (!(USB0.D0FIFOSEL.BIT.CURPIPE != num)) ;
+  while (USB0.D0FIFOSEL.BIT.CURPIPE != num) ;
```

Comment:
this is an 180 degree logic changes, though I think it is correct one. I understand that we are reading and waiting to confirm out num is written. But again, I am not entirely, though it is worth a confirmation.

Reply:
Yes, the modification is correct. The original code was wrong logic.

Reply:
The user manual describes followings.
![image](https://user-images.githubusercontent.com/45088311/126856201-5b71bd91-2315-415e-81eb-b6c3d4df04c1.png)

Reply:
thanks, it is much clearer now. I think the PR is almost ready to merge :)